### ISSUE TITLE
Error during installation

### ISSUE THREAD

**Issue by johgoe** [kyma-incubator/compass#1970]
I got an error during installation on a Kyma 1.24.1 

seems like the sh.ory.oathkeeper.v1alpha1.Rule.spec helm charts of compass are not fit the crds. The kyma cluster without compass works fine.

```
2021/07/31 18:05:53 Processing Kyma components
2021/07/31 18:05:53 ---------------------------
2021/07/31 18:05:53 install component compass
2021/07/31 18:05:53 ---------------------------
2021/07/31 18:05:53 Get Kyma Sources
2021/07/31 18:05:53 Kyma sources available locally.
2021/07/31 18:05:53 Get Kyma Sources...DONE
2021/07/31 18:05:54 Step error:  Details: Helm installation of release "compass" failed: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Rule.spec): unknown field "description" in sh.ory.oathkeeper.v1alpha1.Rule.spec
```

**Comment by desislavaa** [comment]
Hello, we currently support [Kyma v1.21.2](https://github.com/kyma-incubator/compass/blob/master/installation/resources/KYMA_VERSION), but we have planned to update to v1.24 around mid of September. 
Kyma 1.24 comes with updated Ory Oathkeeper version, and we have to adapt our `Rule`s accordingly.

**Comment by StefanCholakov** [comment]
Hello, we already adapted our `Rules` as part of upgrading Kyma to version 1.23.1. Changes can be seen [here:](https://github.com/kyma-incubator/compass/commit/8d2dec9935fdfb8444ad2cc5949cea95a51c20aa)

**Comment by StefanCholakov** [comment]
The issue is fixed after we upgraded Kyma to version 1.24.8
